js/src/modals/ExportAccount/exportStore.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { action, observable } from 'mobx';
+import FileSaver from 'file-saver';
+
+export default class ExportStore {
+  @observable canExport = false;
+  @observable selectedAccount = '';
+  @observable selectedAccounts = {};
+  @observable accountValue = '';
+  @observable inputValue = {};
+
+  constructor (api, accounts, newError, address) {
+    this._api = api;
+    this._accounts = accounts;
+    this._newError = newError;
+    this._address = address;
```

Comment:
I think a variable to toggle between single and multiple exports should be sufficient here. It seems strange to have these two different variables : `inputValue` for multiple exports and `accountValue` for a single one. I think it could be simpler.